lib/Serializer/XmlEncoder.php

```diff
@@ -61,10 +61,9 @@ public function encode(SuiteCollection $suiteCollection): Document
                 $infoEl = $envEl->appendElement($information->getName());
 
                 foreach ($information as $key => $value) {
-                    $valueEl = $infoEl->appendElement('value');
+                    $valueEl = $infoEl->appendTextNode('value', $value);
```

Comment:
note this is just one place - in your original example I think the `echo "Foo & Bar"` was in the error message. As none of what we do here should be passing a literal `&` is is probably fine to replace all `appendElement` with `appendTextNode`

Reply:
hmm I did not spot any other occurence where a node is appended with a value.

all other remaining spots seem to only set attributes. did I miss something?

Reply:
ahh just saw 2 other spots. anything left?